job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/triggers/TriggerContext.groovy

```diff
@@ -154,6 +155,44 @@ class TriggerContext extends AbstractExtensibleContext {
                 allowMembersOfWhitelistedOrgsAsAdmin pullRequestBuilderContext.allowMembersOfWhitelistedOrgsAsAdmin
             }
         }
+
+        NodeBuilder extensionsNodeBuilder = new NodeBuilder()
```

Comment:
And make sure that the XML for the new elements is only generated when the required plugin version is installed, see the `allowMembersOfWhitelistedOrgsAsAdmin` above for an example.